web/src/components/text-input/styles.ts

```diff
@@ -0,0 +1,91 @@
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../tokens'
+import { getColorDefault, getColorHover } from '../checkbox/styles'
+
+export default createUseStyles({
+  textField: {
+    fontFamily: tokens.fontFamily,
+    borderRadius: ({ component }) => {
+      return component === 'input'
+        ? tokens.borderRadiusFormfieldInput
+        : tokens.borderRadiusFormfieldTextarea
+    },
+    borderWidth: tokens.borderWidthFormfield,
+    borderStyle: 'solid',
+    color: tokens.colorTextPrimary,
+    resize: ({ allowResize }) => (allowResize ? 'vertical' : 'none'),
+    width: '100%',
+    backgroundColor: ({ isDisabled, isReadOnly }) => {
+      if (isDisabled) return tokens.colorBackgroundFormfieldDisabled
+      if (isReadOnly) return tokens.colorBackgroundFormfieldReadonly
+      return tokens.colorBackgroundFormcontrolDefault
+    },
+    cursor: ({ isDisabled }) => {
+      if (isDisabled) return 'not-allowed'
+      return undefined
+    },
+    height: ({ component, maxLength }) => {
+      if (component === 'input') return tokens.sizeFormfieldInput
+      if (maxLength) {
+        return '92px'
+      }
+      return '60px'
+    },
+    minHeight: ({ minHeight }) => minHeight,
+    maxHeight: ({ maxHeight }) => maxHeight,
+    boxSizing: 'border-box',
+    padding: () => {
```

Comment:
here can be a string instead of a function

Reply:
done